test/ServerComparison.TestSites/ServerComparison.TestSites.csproj

```diff
@@ -5,9 +5,14 @@
     <RuntimeIdentifiers>win7-x86;win7-x64;linux-x64;osx-x64</RuntimeIdentifiers>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
+  <ItemGroup Condition="'$(OS)' == 'Windows_NT' AND ('$(ANCMVersion)' == 'V2' Or '$(ANCMVersion)' == '')">
     <PackageReference Include="Microsoft.AspNetCore.AspNetCoreModule" Version="$(MicrosoftAspNetCoreAspNetCoreModulePackageVersion)" />
```

Comment:
So AspNetCoreModule is always the latest version, not necessarily the one we ship?

Reply:
Potentially. We may want to remove the M.A.AspNetCoreModule nuget package and make it M.A.AspNetCoreModuleV2. I can make that change in a separate PR.